### PR TITLE
[SYCL] Add tests for code location print in case of exception in execution

### PR DESCRIPTION
**Pull request by KseniyaTikhomirova**

### CODE REVIEW

**Comment by KseniyaTikhomirova** [comment]
tests for https://github.com/intel/llvm/pull/8101. Should not be merged before.

File: SYCL/Tracing/code_location_queue_parallel_for.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  try {
+    Queue.parallel_for<class E2ETestKernel>(
+        sycl::nd_range<1>{MaxWISizes.get(0), 2 * MaxWISizes.get(0)},
+        [](sycl::id<1> idx) {});
+  } catch (...) {
+    ExceptionCaught = true;
+  }
+  Queue.wait();
+  return !ExceptionCaught;
+}
```
Comment (againull) [inline SYCL/Tracing/code_location_queue_parallel_for.cpp]: NIT: Please fix "no newline at end of file"

**Review by againull — APPROVED** [review]

File: SYCL/Tracing/code_location_queue_parallel_for.cpp (hunk 1 above)
Comment (KseniyaTikhomirova) [reply SYCL/Tracing/code_location_queue_parallel_for.cpp]: should be fixed now

**Comment by KseniyaTikhomirova** [comment]
@intel/llvm-gatekeepers this commit is ready for merge

**Pull request merged**